designcompose/src/main/java/com/android/designcompose/squoosh/SquooshTreeBuilder.kt

```diff
@@ -142,6 +143,7 @@ internal fun resolveVariantsRecursively(
     useLocalStringRes: Boolean?,
 ): SquooshResolvedNode? {
     if (!customizations.getVisible(v.name)) return null
+    customizations.getVisibleState(v.name)?.let { if (!it.value) return null }
```

Comment:
This will cause a full recomposition, but that's probably OK until we have performance data showing otherwise.

For text, we could do something clever and allow text changes to only invalidate rendering if we know they can't change layout (ie the text node size doesn't change based on text content).

Reply:
Yea that makes sense. For visibility we would need to determine if changing visibility changes the layout due to its parent being autolayout, but I wanted to keep this change simple.